### PR TITLE
Add information about the project and how to play the game in the REA…

### PR DESCRIPTION
**Pull request by Phoebe-Khokgawe**
…DME file

### CODE REVIEW

File: portfolio/frontend/README.md [hunk 1]
```diff
@@ -1,4 +1,35 @@
-# Getting Started with Create React App
+# SGONKS - Trading Game
+
+SGONKS is a trading game created with Node.js with Server in Java.
```
Comment (leonnemets) [inline portfolio/frontend/README.md:3]: with a java backend

File: portfolio/frontend/README.md [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+
+## Table of Contents
+
+- [About the Project](#general-info)
```
Comment (leonnemets) [inline portfolio/frontend/README.md:7]: did you forget the corresponding general-info  tag?

File: portfolio/frontend/README.md (hunk 1 above)
Comment (Phoebe-Khokgawe) [reply portfolio/frontend/README.md]: fix here 9acb7ad04cc0d22ebd09636ced7b845c55048b93

File: portfolio/frontend/README.md (hunk 2 above)
Comment (Phoebe-Khokgawe) [reply portfolio/frontend/README.md:7]: fix here 9acb7ad04cc0d22ebd09636ced7b845c55048b93

**Review by leonnemets — COMMENTED** [review]
Nice

**Review by leonnemets — APPROVED** [review]

**Comment by roonil-wazlib** [comment]
@googlebot I consent.

**Review by lin8231 — APPROVED** [review]

**Pull request merged**